### PR TITLE
12131 Renamed Document Delivery label for staff view in review and confirm.

### PR DESCRIPTION
**Pull request by lambert-alex**
*Issue #:* /bcgov/entity#12131

*Description of changes:*

- Changed the label for staff review and confirm view of document delivery from "User Account" to "Completing Party". 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Review by severinbeauvais — APPROVED** [review]
Looks good but you should update the app version, too :)

File: src/components/common/DocumentsDelivery.vue [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
         <v-row v-if="isRoleStaff" no-gutters class="mt-6">
           <v-col cols="3" class="px-0">
-            <label :class="{ 'error-text': documentDeliveryInvalid }"><strong>User Account</strong></label>
+            <label :class="{ 'error-text': documentDeliveryInvalid }"><strong>Completing Party</strong></label>
```
Comment (lewischenstudio) [inline src/components/common/DocumentsDelivery.vue:23]: While a staff can access the filing, a registered SP/GP user can also access it. I think we need a conditional rendering for this component. Furthermore, I noticed "User Account" appears in two places in this file, and the other one is in line 43. Can you verify with the UI/UX team whether either or both of them needs the change?

Comment (lambert-alex) [reply src/components/common/DocumentsDelivery.vue:23]: Yes the "User Account" on line 43 is what renders the SP/GP view of the Document delivery component. The ticket said only change for the Completing party. However, I will verify with the UI/UX team. That would be Roy or Yui right?

Comment (lewischenstudio) [reply src/components/common/DocumentsDelivery.vue:23]: Yes, Roy and Yui are in the UI/UX team.

File: src/App.vue [hunk 2]
```diff
@@ -161,7 +161,7 @@ import * as Dialogs from '@/dialogs/'
 import { AuthServices } from '@/services/'
 import { CommonMixin, DateMixin, FilingTemplateMixin, LegalApiMixin } from '@/mixins/'
 import { FilingDataIF, ActionBindingIF, ConfirmDialogType, FlagsReviewCertifyIF, FlagsCompanyInfoIF,
-  AlterationFilingIF, FirmChangeIF, FirmConversionIF } from '@/interfaces/'
+  AlterationFilingIF, ChangeFilingIF, ConversionFilingIF } from '@/interfaces/'
```
Comment (severinbeauvais) [inline src/App.vue:164]: This PR includes commits that are already in main branch. Was there an issue while rebasing?

Comment (lambert-alex) [reply src/App.vue:164]: Yes there was, should I roll it back a commit to fix this?

Comment (lambert-alex) [reply src/App.vue]: Its strange I found some other people have had this issue and I found a workaround that involved changing a the base branch to something other than main and then change the branch back to main, then GitHub registered the change. It seems to have worked.

File: src/components/common/DocumentsDelivery.vue (hunk 1 above)
Comment (lambert-alex) [reply src/components/common/DocumentsDelivery.vue:23]: Looks like you were right Lewis, thanks for that awesome catch!

**Comment by severinbeauvais** [comment]
- [ ] conflicts need to be resolved
- [ ] Lewis needs to approve (if satisfied)
- [ ] let's see if Lewis has permissions to merge this PR

**Review by severinbeauvais — APPROVED** [review]
LGTM

**Review by lcycstudio — APPROVED** [review]
Everything looks good.

**Review by lewischenstudio — APPROVED** [review]
Everything looks good.

**Comment by lewischenstudio** [comment]
> * [x]  conflicts need to be resolved
> * [ ]  Lewis needs to approve (if satisfied)
> * [ ]  let's see if Lewis has permissions to merge this PR

I think I don't permissions to merge this PR, but I have permission to close it with comment.

**Pull request merged**